DEA_products/DEA_Wetlands_Insight_Tool.ipynb

```diff
@@ -0,0 +1,2403 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    # rasterise the shapefile where poly is the vector data and pv is the xarray template
</pre><p>Some comments start with capitals, some don't - would be nice to standardise (most other notebooks start comments with capitals)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GeoscienceAustralia/dea-notebooks/pull/1102/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>